### PR TITLE
Tanium Ask playbook - Added delete context for TaniumQuestion key

### PR DESCRIPTION
**Pull request by teizenman**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/29525

## Description
The playbook queried the context at `Tanium.Question` and this has caused redundant requests to the API.

### CODE REVIEW

File: Packs/Tanium/Playbooks/playbook-Tanium_-_Ask_Question.yml [hunk 1]
```diff
@@ -163,14 +209,18 @@ inputs:
   value: {}
   required: false
   description: ""
+  playbookInputQuery:
 - key: parameters
   value: {}
   required: false
   description: ""
+  playbookInputQuery:
```
Comment (ronykoz) [inline Packs/Tanium/Playbooks/playbook-Tanium_-_Ask_Question.yml:217]: why do we have this twice?

**Review by ronykoz — APPROVED** [review]
so why are we deleting the context after the tanium.question?

File: Packs/Tanium/Playbooks/playbook-Tanium_-_Ask_Question.yml (hunk 1 above)
Comment (teizenman) [reply Packs/Tanium/Playbooks/playbook-Tanium_-_Ask_Question.yml:217]: Its different keys.

**Comment by teizenman** [comment]
> so why are we deleting the context after the tanium.question?

Its not after, this is just how the YML is arragned, the first task in no. 5 (`DeleteContext`) and then goes to no. 1 (`tn-ask-question`)

**Pull request closed without merging**